handler.go

```diff
@@ -105,3 +107,20 @@ func ToWriter(w io.Writer) ResponseHandler {
 		return err
 	})
 }
+
+// ToFile writes the response body to file.
+func ToFile(name string) ResponseHandler {
+	return func(res *http.Response) error {
+		err := os.MkdirAll(filepath.Dir(name), 0777)
```

Comment:
Creating a directory to put files into is always technically a race, since any other process on the machine can do something else behind your back. I usually just explicitly ignore the error here, because if it worked or not will be determined by the subsequent call to os.Create.